docs/backup_and_restore.rst

```diff
@@ -123,67 +105,216 @@ Restoring
 
-To perform a restore, you must already have a backup archive. Provide its
-filename in the following command:
+Restoring Data on an Existing SecureDrop Installation
+''''''''''''''''''''''''''''''''''''''''''''''''''''' 
+
+To restore an existing instance to a previous state, run the command:
 
 .. code:: sh
 
-   ./securedrop-admin restore sd-backup-2017-07-22--01-06-25.tar.gz
+   ./securedrop-admin restore sd-backup-2020-07-22--01-06-25.tar.gz
 
-Make sure to replace ``sd-backup-2017-07-22--01-06-25.tar.gz`` with the filename
+Make sure to replace ``sd-backup-2020-07-22--01-06-25.tar.gz`` with the filename
 for your backup archive. The backup archives are located in
 ``install_files/ansible-base``.
 
-Once the restore is done, the *Application Server* will use the original
-*Source Interface* and *Journalist Interface* Onion URLs. You will need
-to update the corresponding files on the *Admin Workstation*:
+This command restores submissions, source and journalist accounts, and Tor
+configuration (*Source* and *Journalist Interface* onion URLs) to their state at the 
+time of the backup.
 
-* ``app-source-ths``
-* ``app-journalist-aths``
-* ``app-ssh-aths``
+.. _migrating:
 
-Once SSH access to the servers has been established (or if using SSH over
-local network), Onion URLs for the *Source Interface* and *Journalist Interfaces*
-can be fetched using the installer:
+Migrating
+---------
 
-.. code:: sh
+.. note:: These steps require physical access to the servers. 
+         
+Moving a SecureDrop installation to new hardware consists of:
 
-   ./securedrop-admin install
+  - Backing up the existing installation and preserving its Tor credentials from the *Admin Workstation* 
+  - Installing the same version of SecureDrop on the new hardware;
+  - Restoring the backup to the new installation;
+  - Repairing Tor credentials for the new installation. 
 
-Then rerun ``./securedrop-admin tailsconfig`` to update the *Admin Workstation*
-to use the restored Onion URLs again. See :doc:`configure_admin_workstation_post_install`
-for detailed instructions.
+#. :ref:`Back up the existing installation <backing_up>`.
 
-Migrating
----------
+#. **Preserve important credentials**: Before re-installing SecureDrop, you 
+   will need to preserve some important credential files from the old 
+   installation on the *Admin Workstation*. They should be temporarily saved
+   somewhere in the ``~/Persistent/`` directory. 
+
+   .. code:: sh
+
+       mkdir ~/Persistent/sd-restore-creds
+
+   Copy in the following files from ``~/Persistent/securedrop/install_files/ansible-base``:
+
+   **Instances using v2 onion services:**
+   
+   - ``app-source-ths``
+   - ``app-journalist-aths``
+   - ``app-ssh-aths``
+
+
+   **Instances using v3 onion services:**
+
+   - ``app-sourcev3-ths``
+   - ``app-ssh.auth_private``
+   - ``app-journalist.auth_private``
+   -  ``tor_v3_keys.json``
+
+
+   Once you have preserved those files, you may either delete or rename the 
```

Comment:
I'm not sure we even want to enumerate deletion as an option at this stage, in case any contents of the directory are later needed for diagnostics. I would suggest just offering the ``securedrop.old`` step here.